modules/reactor/test/tests/functions/multi_control_drum_function/positions_meta.data

```diff
@@ -1,6 +1,6 @@
 6.06218 3.5 0.0
--5.78862e-15 7 0.0
```

Comment:
Why is this file changed here?

Reply:
This is a "zero" calculated by the new `sum(Point)`.

Reply:
Thanks, I see.

But now that I think about the problem: why bother changing it in the repo?  MPI_Reduce operations don't guarantee any particular operation ordering, and floating point addition isn't associative so you can't expect the results to go unchanged if you reorder operations, so even if we hadn't changed which `sum()` to use, we'd still expect the final result to change by some multiple of machine precision even if we re-run it on the same MPI implementation on the same machine on the same number of (more than two) processors.  (and we'd expect it to change more if any of those things change).

So I'm worried: you bothering to commit this file suggests that O(1e-14) changes here are really important ... are they?  If that's the case then we must be doing something wrong elsewhere, because O(1e-14) changes here are unavoidable.

Reply:
I agree with you. I will remove this file and add into .gitignore. It is generated for users but not useful for this regression test.